plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2801,7 +2801,7 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
                         Optional.empty()), // forAnalyze does not affect stats
                 handle -> {
                     Table icebergTable = catalog.loadTable(session, handle.getSchemaTableName());
-                    return TableStatisticsReader.getTableStatistics(typeManager, session, handle, icebergTable);
+                    return TableStatisticsReader.getTableStatistics(typeManager, session, handle, icebergTable, fileSystemFactory.create(session.getIdentity(), icebergTable.io().properties()));
```

Comment:
this line became long. 
we put all arguments on one line, or each on separate line:

```java
return TableStatisticsReader.getTableStatistics(
        typeManager,
        session,
        handle,
        icebergTable,
        fileSystemFactory.create(session.getIdentity(), icebergTable.io().properties()));
```